docs/backup-restore.md

```diff
@@ -0,0 +1,109 @@
+# Backup and Restore TiDB Cluster
+
+## Overview
+
+TiDB Operator supports two kinds of backup:
+
+* [Full backup](#full-backup)(scheduled or ad-hoc) via [`mydumper`](https://www.pingcap.com/docs/dev/reference/tools/mydumper/), which helps you logical backup the TiDB cluster.
+* [Incremental backup](#incremental-backup) via [`TiDB-Binlog`](https://www.pingcap.com/docs/dev/reference/tools/tidb-binlog/overview/), which helps you synchronize the data in the TiDB cluster to other database or backup the data at real-time.
+
+Currently, tidb-operator only supports automatic [restore operation](#restore) for full backup taken by `mydumper`. Restore the backup data captured by `TiDB-Binlog` requires human intervention.
+
+## Full backup
+
+Full backup using `mydumper` to take the logical backup of TiDB cluster. The backup job will create a PVC to store backup data.
+
+By default, the backup uses PV to store the backup data. You can also store the backup data to [Google Cloud Storage](https://cloud.google.com/storage/) bucket or [Ceph Object Storage](https://ceph.com/ceph-storage/object-storage/) by changing the configuration. This way the PV temporarily stores backup data before it is placed in object storage. Refer to [TiDB cluster Backup configuration](./references/tidb-backup-configuration.md) for full configuration guide of backup and restore.
+
+You can either setup a scheduled full backup or take a full backup in ad-hoc manner.
+
+### Scheduled full backup
+
+Scheduled full backup is created along side the TiDB cluster, and it runs periodically like the crontab job.
+
+To configure a scheduled full backup, modify the `scheduledBackup` section in the `charts/tidb-cluster/values.yaml` of the tidb cluster:
+
+* Set `scheduledBackup.create` to `true`
+* Set `scheduledBackup.storageClassName` to the PV storage class name used for backup data
+
+> **Note:** You must set the scheduled full backup PV's [reclaim policy](https://kubernetes.io/docs/tasks/administer-cluster/change-pv-reclaim-policy) to `Retain` to keep your backup data safe.
+
+* Configure `scheduledBackup.schedule` in the [Cron](https://en.wikipedia.org/wiki/Cron) format to define the scheduling
+* `scheduledBakcup.user` and `scheduledBackup.password` must be set to the correct user which has the permission to read the database to be backuped.
```

Comment:
operator uses `scheduledBackup.secretName` to store the username and password now.

user can create the secret by:
```
kubectl create secret generic backup-secret --from-literal=user=root --from-literal=password=<password>
 ```